sumpy/expansion/local.py

```diff
@@ -682,6 +668,42 @@ def translate_from(self, src_expansion, src_coeff_exprs, src_rscale,
         logger.info("building translation operator: done")
         return result
 
+    def loopy_translate_from(self, src_expansion):
+        from sumpy.expansion.multipole import VolumeTaylorMultipoleExpansionBase
+
+        if isinstance(src_expansion, VolumeTaylorMultipoleExpansionBase):
+            ncoeff_src = self.m2l_preprocess_multipole_nexprs(src_expansion)
+            ncoeff_tgt = self.m2l_postprocess_local_nexprs(src_expansion)
+
+            icoeff_src = pymbolic.var("icoeff_src")
+            icoeff_tgt = pymbolic.var("icoeff_tgt")
+            domains = [f"{{[icoeff_tgt]: 0<=icoeff_tgt<{ncoeff_tgt} }}"]
+
+            src_coeffs = pymbolic.var("src_coeffs")
+            m2l_translation_classes_dependent_data = pymbolic.var("data")
+
+            if self.use_preprocessing_for_m2l:
```

Comment:
That flag name is no longer correct.

Reply:
Done in c333c313f77d9451cffbdd443db8ee8fbdcdc43d